site/en/docs/privacy-sandbox/attribution-reporting/system-overview/index.md

```diff
@@ -119,18 +119,46 @@ While this API is in testing, your code must confirm the API is available and se
 <figcaption><strong>Figure 1</strong>.</figcaption>
 </figure>
 
-### Attribution sources (publisher's website) {: #attribution-sources}
 
-An _[attribution source](/docs/privacy-sandbox/attribution-reporting/register-attribution-source/)_
-is an ad-related event (a click or view), to which an ad tech can attach the
-following kinds of information:
 
-*  Contextual reporting data, such as the ad creative ID, information about the
-   campaign, or geography.
-*  A conversion destination, as in the site where you expect the user will convert.
+### Attribution event flow
+
+Imagine a publisher site that displays ads. Each advertiser or ad tech provider wants to learn about interactions with their ads, and attribute conversions. Reports—both event-level and aggregatable—would be generated as follows:
```

Comment:
Attribute conversions to what? I can guess, but as it is the sentence feels incomplete.